src/PageUpButton.jsx

```diff
@@ -0,0 +1,25 @@
+import { Icon, IconButton } from 'rsuite';
+import { FlexboxGrid } from 'rsuite';
+
+const PageUpButton = ({isVisibleButton}) => {
+
+    const scrollToTop = () => 
+      window.scrollTo({
+        top: 0,
+        behavior: "smooth"
+      })
+    
+    return (
+      <FlexboxGrid className="page-up-button">
+        <IconButton
+          style={{display: !isVisibleButton && "none"}}
```

Comment:
move this style into className... as we agreed before..

Reply:
sorry i forgot ...
i remember you showed it BUT i dont remember what exactly you showed me 👉👈 🥺

Reply:
done